dali/dfuplus/dfuplus.cpp

```diff
@@ -367,15 +367,29 @@ bool CDfuPlusHelper::variableSpray(const char* srcxml,const char* srcip,const ch
 
     const char* encoding = globals->queryProp("encoding");
     const char* rowtag = globals->queryProp("rowtag");
-    if(stricmp(format, "xml") == 0)
+    const char* rowpath = globals->queryProp("rowpath");
+    if(strieq(format, "json"))
```

Comment:
I prefer strieq because the return boolean is more intuitive and tends to produce more readable code.  But I also sometimes take the surrounding code into consideration.  Maybe the stricmp was far enough away in this case. :)